clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -638,8 +625,8 @@ else if (error == Errors.CLUSTER_AUTHORIZATION_FAILED)
                             "topic-partition may not exist or the user may not have Describe access to it",
                         batch.topicPartition);
                 } else {
-                    log.warn("Received invalid metadata error in produce request on partition {} due to {}. Going " +
-                            "to request metadata update now", batch.topicPartition, error.exception().toString());
+                    // log.warn("Received invalid metadata error in produce request on partition {} due to {}. Going " +
```

Comment:
Should we revert this? Was `warn` too high?

Reply:
I think it's fine, I think I was seeing it a lot because I was running tests using JDK 8 but I had built with JDK 11 , so the metadata calls were constantly erroring out. Normally I would expect this to log a few times and then resolve